polaris-react/.storybook/GridOverlay/GridOverlay.css

```diff
@@ -15,8 +15,8 @@
   opacity: 0.25;
   pointer-events: none;
 }
-
-@media screen and (min-width: 768px) {
+/* #{$p-breakpoints-sm-up} */
+@media (min-width: 768px) {
```

Comment:
Added a comment of the associated breakpoint alias to allow the media condition to show up in search results.